backend/GymBeam/CommandHandlers/DeleteUserCommandHandler.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Domain;
+using GymBeam.Commands;
+using GymBeam.Exceptions;
+using MediatR;
+
+namespace GymBeam.CommandHandlers
+{
+    public class DeleteUserCommandHandler : IRequestHandler<DeleteUserCommand, Unit>
+    {
+        private readonly IRepository _repository;
+
+        public DeleteUserCommandHandler(IRepository repository)
+        {
+            _repository = repository;
+        }
+
+        public Task<Unit> Handle(DeleteUserCommand request, CancellationToken cancellationToken)
+        {
+            User user;
+            try
+            {
+                user = _repository.GetById<User>(request.UserId);
+                _repository.Delete(user);
```

Comment:
there should be check
if user is null: throw domain exception. Please remove this try catch from here

Reply:
fix it everywhere else respectively